### DISCUSSION TITLE
Select the social network you want in your profile.

### DISCUSSION THREAD

**Discussion by rodukov** [community/community#33989] [Other Feature Feedback, Questions, & Ideas]
🔸 At the moment, **you can only select Twitter,** and it is not clear why. 
I would recommend giving people a choice on the default social network.

**Comment by ad-olfo** [comment]
Giving us new fields to add more social links, like LinkedIn or Mastodon, will be definitely a nice add.

**Comment by derekprior** [comment]
Hey folks, I just wanted to give you a heads up that we are working on this. It'll ship when it's ready, but with holiday season vacations coming up that won't be for a bit yet. Thanks for your patience.

**Reply by ad-olfo** [reply]
Great news! Thank you for your follow-up Derek, happy holidays 😉

**Comment by Forsann** [comment]
T
